ADAL/src/cache/ADTokenCacheItem+Internal.m

```diff
@@ -257,8 +258,13 @@ - (void)logMessage:(NSString*)message level:(ADAL_LOG_LEVEL)level correlationId:
     {
         [ADLogger log:level context:self correlationId:correlationId isPii:YES
                format:@"%@", _tombstone];
-        [ADLogger log:level context:self correlationId:correlationId isPii:NO
-               format:@"{\n\tresource: %@\n\tclientId: %@\n\tauthority:%@\n}", _resource, _clientId, _authority];
+        [ADLogger log:level context:self correlationId:correlationId isPii:YES
+               format:@"\n\tresource: %@", _resource];
+        [ADLogger log:level context:self correlationId:correlationId isPii:YES
+               format:@"\n\tclientId: %@", _clientId];
+        BOOL isKnownHost = [ADAuthorityUtils isKnownHost:[_authority adUrl]];
+        [ADLogger log:level context:self correlationId:correlationId isPii:!isKnownHost
```

Comment:
same here: only authority host can be logged if it's a known host, full authority url can contain the tenant ID :(

Reply:
fixed